db/1.8/modules/ROOT/nav.adoc

```diff
@@ -1,7 +1,7 @@
 .xref:index.adoc[Database Connector]
 * xref:index.adoc[About Database Connector]
-* xref:database-connector-design-center.adoc[Database Design Center Configuration]
 * xref:database-connector-studio.adoc[Database Studio Configuration]
-* xref:database-connector-xml-maven.adoc[Database XML and Maven Support]
-* xref:database-connector-examples.adoc[Database Examples]
 * xref:database-documentation.adoc[Database Connector Reference]
+* xref:database-connector-examples.adoc[Database Examples]
```

Comment:
I usually put Connector in the title  Database Connector Examples, Database Connector XML and Maven Support, etc
And then for the Design Center topic, Use Flow Designer to Configure Database Connector or something like that.